.gitignore

```diff
@@ -33,7 +33,6 @@ Engine/resource/version.aps
 *.o
 *.d
 common.a
-ags
```

Comment:
Ah! It was hiding all changes in OSX/xcode/ags and driving me mad.  I readded and made more specific in b3e0f7e0573340322e68178373fbcb3d855920c1